dryville_story.md

```diff
@@ -16,3 +16,16 @@ From the founding day of Dryville, you need water. Drinking water is your first
 
 In your free time go ahead and start digging a hole in the ground for a [well](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-wells). If you dig deep enough you might hit the [water table](https://www.usgs.gov/special-topic/water-science-school/science/aquifers-and-groundwater), where there will be standing water. You'll be able to lower a bucket down to get the available [groundwater](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater). Things are starting out great -- you've already started using surface water (the creek and pond), ground water (your well), and you've even created your own water-storage system (the reservoir). So you now have a reliable source for your water needs. Time to relax? Not yet.
 
+## Getting Water to Your Homes
+
+Since you appointed yourself mayor Horace of Dryville, you naturally chose the best spot for your home — on top of the hill. Great view, but it sure is a pain lugging pails of water (at [over 8 pounds a gallon](https://www.usgs.gov/special-topic/water-science-school/science/water-density), your 100 gallons of water per day gets heavy in a hurry!) from the creek up the hill all day long. And now all your best buddies and neighbors are beginning to move in. Many of them are building on hills, too, so what you need now is a "water-supply distribution system" to get water to everyone's homes. The way to do this is to lay a system of pipes (which you make from clay from the creek bed) from the creek to each house.
```

Comment:
Nice choice. I like this WSC story :heart:

Reply:
Tried to keep it fun, but also relevant :)